store/tikv/2pc.go

```diff
@@ -135,6 +147,27 @@ func newTwoPhaseCommitter(txn *tikvTxn, connID uint64) (*twoPhaseCommitter, erro
 	if len(keys) == 0 {
 		return nil, nil
 	}
+
+	for _, pair := range txn.contracts {
+		mutation, ok := mutations[string(pair.key)]
+		if !ok {
+			log.Info(" something is wrong... contract exists but no mutation?", pair)
```

Comment:
Should panic here.

Reply:
It's better to add a TODO here.

Reply:
I think log is easy to be neglected, why panic is not a good choice for now?

Reply:
Because we may have false negative, we think some mutation violate the contract, while the contract itself is wrong @winkyao